persistence-api/src/main/java/io/stargate/db/datastore/DataStore.java

```diff
@@ -80,9 +80,9 @@ default void waitForSchemaAgreement() {
       if (isInSchemaAgreement()) {
         return;
       }
-      Uninterruptibles.sleepUninterruptibly(100, TimeUnit.MILLISECONDS);
```

Comment:
Maybe the is should be configurable? Using something like a `stargate.schema_agreement_timeout` setting.

Reply:
I agree, but it does not look strictly related to this ticket. Maybe we can do it in a separate JIRA task? (I can create one)